CMake: cannot find octomap lib

#### Environment
* DART master, 6.8.0 (but the same applies to release-6.7)
* Ubuntu 18.04 and Arch Linux

When trying to compile the latest DART (master or release-6.7) without octomap installed in my computer, I get the following:

```sh
/usr/bin/ld: cannot find -loctomap
collect2: error: ld returned 1 exit status
make[2]: *** [dart/CMakeFiles/dart.dir/build.make:2116: lib/libdart.so.6.8.0] Error 1
make[1]: *** [CMakeFiles/Makefile2:355: dart/CMakeFiles/dart.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
```

It makes sense, because this line in [dart/CMakeLists.txt](https://github.com/dartsim/dart/blob/master/dart/CMakeLists.txt#L138) tries to link to `octomap` and of course if it is not installed it cannot link to it.

Is this a known issue or am I missing something?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -20,7 +20,7 @@
 
   * Changed to use GNUInstallDirs for install paths: [#1241](https://github.com/dartsim/dart/pull/1241)
   * Fixed not failing for missing required dependencies: [#1250](https://github.com/dartsim/dart/pull/1250)
-  * Fixed linking to octomap when not imported: [#1253](https://github.com/dartsim/dart/pull/1253)
+  * Fixed attempting to link octomap when not imported: [#1253](https://github.com/dartsim/dart/pull/1253)
 
 ### [DART 6.7.3 (2019-02-19)](https://github.com/dartsim/dart/milestone/51?closed=1)
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -20,6 +20,7 @@
 
   * Changed to use GNUInstallDirs for install paths: [#1241](https://github.com/dartsim/dart/pull/1241)
   * Fixed not failing for missing required dependencies: [#1250](https://github.com/dartsim/dart/pull/1250)
+  * Fixed linking to octomap when not imported: [#1253](https://github.com/dartsim/dart/pull/1253)
 
 ### [DART 6.7.3 (2019-02-19)](https://github.com/dartsim/dart/milestone/51?closed=1)
 

--- a/dart/CMakeLists.txt
+++ b/dart/CMakeLists.txt
@@ -135,8 +135,10 @@ target_link_libraries(dart
     Boost::boost
     Boost::system
     Boost::filesystem
-    octomap
 )
+if (TARGET octomap)
+  target_link_libraries(dart PUBLIC octomap)
+endif()
 if(NOT MSVC)
   target_link_libraries(dart PUBLIC Boost::regex)
 endif()

